.changeset/stupid-lizards-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Deprecated: `context.fix` in favour of the `fix` callback
```

Comment:
[suggest] Minor. Adding `report()` may be more understandable for users. For example,

```suggestion
Deprecated: `context.fix` in favour of the `fix` callback in `report()`
```

Reply:
```suggestion
Deprecated: `context.fix` usage in favour of recommending to pass a `fix` callback to `report()`
```

Reply:
Sounds good 👍🏼